src/monitoring_service/cli.py

```diff
@@ -53,6 +56,30 @@ def setup_logging(log_level: str) -> None:
     )
 
 
+def check_gas_reserve(web3: Web3, private_key: str) -> bool:
+    """ Check periodically for gas reserve in the account """
+    gas_price = web3.eth.gasPrice
+    gas_limit = GAS_REQUIRED_FOR_MS_MONITOR
```

Comment:
I don't see the need to this variable, you can just inline this.

Reply:
I tried, but the lime becomes too long when factor 10 is introduced as a default variable. Looks like this now:
   ` estimated_required_balance = gas_limit * gas_price * DEFAULT_GAS_BUFFER_FACTOR`